org.mixedrealitytoolkit.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -232,12 +232,11 @@ public override bool isHoverActive
                         }
 #pragma warning restore CS0612
 #pragma warning restore CS0618
-                        else
+                        // Attempt palm facing away check if the interactor is associated with a hand.
+                        else if (handedness != InteractorHandedness.None &&
+                            (XRSubsystemHelpers.HandsAggregator?.TryGetPalmFacingAway(handedness.ToXRNode(), out isPalmFacingAway) ?? true))
```

Comment:
should this be `?? false`? If `HandsAggregator` is null, then we won't have a meaningful value for `isPalmFacingAway`

Reply:
@keveleigh @AMollis I would tend to agree, but I believe @ms-RistoRK was leaving the [existing logic](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/blob/main/org.mixedrealitytoolkit.input/Interactors/Ray/MRTKRayInteractor.cs#L161) of `?? true` in place - not sure why that was originally chosen if HandsAggregator was null, though?